src/practices/LanguageIndependent/TimeToSolvePractice.ts

```diff
@@ -0,0 +1,55 @@
+import moment from 'moment';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { GitServiceUtils } from '../../services/git/GitServiceUtils';
+import { DxPractice } from '../DxPracticeDecorator';
+import { IPractice } from '../IPractice';
+
+@DxPractice({
+  id: 'LanguageIndependent.TimeToSolve',
+  name: '',
+  impact: PracticeImpact.medium,
+  suggestion: '',
+  reportOnlyOnce: true,
+  url: 'https://dxkb.io/p/pull-requests',
+  dependsOn: { practicing: ['LanguageIndependent.DoesPullRequests'] },
+})
+export class TimeToSolvePractice implements IPractice {
+  async isApplicable(): Promise<boolean> {
+    return true;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (ctx.fileInspector === undefined || ctx.collaborationInspector === undefined) {
```

Comment:
```suggestion
    if (!ctx.fileInspector || !ctx.collaborationInspector) {
```
do not copy&paste only... try to improve the copied code